instrumentation-api-incubator/src/main/jflex/SqlSanitizer.jflex

```diff
@@ -122,13 +122,48 @@ WHITESPACE          = [ \t\r\n]+
     /** @return true if all statement info is gathered */
     boolean handleNext() {
       return false;
-     }
+    }
+
+    /** @return true if all statement info is gathered */
+    boolean handleOperationTarget(String target) {
+      return false;
+    }
+
+    boolean waitingOperationTarget() {
+      return false;
+    }
 
     SqlStatementInfo getResult(String fullStatement) {
       return SqlStatementInfo.create(fullStatement, getClass().getSimpleName().toUpperCase(java.util.Locale.ROOT), mainIdentifier);
     }
   }
 
+  private static abstract class CompositeOp extends Operation {
```

Comment:
In my opinion `CompositeOp` isn't a good name for this class. I'd expect a composite operation to combine 2 or more operations in some way which this class does not do. Perhaps `DdlOperation` would be better?

Reply:
Renamed.